fixtures/wecan/wkl-beam02.json

```diff
@@ -0,0 +1,72 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "WKL-BEAM02",
+  "categories": ["Color Changer", "Dimmer"],
+  "meta": {
+    "authors": ["Zu Kim"],
+    "createDate": "2024-10-01",
+    "lastModifyDate": "2024-10-01"
+  },
+  "links": {
+    "manual": [
+      "https://downloads.monoprice.com/files/manuals/612210_Manual_140327.pdf"
```

Comment:
Something looks fishy.  Is this really the manual for the WECAN WKL-BEAM02?  It doesn't mention that brand or product name anywhere.  The weight and power are slightly different, and number of channels is wrong.  The case looks similar but the LED display is on the opposite side.  I don't see any links from WECAM to Monoprice, or vice versa, to suggest a relationship here.